README.md

```diff
@@ -24,4 +27,6 @@ The program expects target url to take HTML(and check for dead links) from.
 }
 ```
 
-It is ok to use YML/XML or any other format considering the test verifying this format is availabl
+It is ok to use YML/XML or any other format (not only JSON) considering the test verifying this format is available.
+
+*N.B.: it worth to think of various edge cases related to CLI usage, potential connection and/or content issues and cover them in your code*
```

Comment:
@totus edge cases are good candidates for further(2, 3, etc) steps. Lets not make initial task too complex.

Reply:
So the suggestion is to skip 32th line for now. The rest looks ok!